package.json

```diff
@@ -47,6 +47,7 @@
     "babel-plugin-module-resolver": "4.1.0",
     "big.js": "6.2.1",
     "bundlesize": "0.18.1",
+    "cross-spawn": "7.0.3",
```

Comment:
This seems to be pretty popular, some other dependency must already require the exact same version because it didn't modify the lock file at all when this was added.

I think it'll help if someone is developing on windows and tries to build the project.